spec/parsers/moov_parser_spec.rb

```diff
@@ -94,6 +94,7 @@ def deep_print_atoms(atoms, output, swimlanes = [])
     expect(result.format).to eq(:mov)
     expect(result.width_px).to eq(1920)
     expect(result.height_px).to eq(1080)
+    expect(result.codecs.first).to eq('apcn')
```

Comment:
Shouldn't we check the whole array of codecs we just parsed?

Reply:
Yes, get that, was just referring to the expectation:

```suggestion
    expect(result.codecs).to eq(['apcn'])
```

Otherwise, it feels the test is only concerned with the first codec, which is not. Matching against the whole array better stands for the point, imho.

Reply:
Yes I get your point! Committed the change!

Reply:
Awesome! 👌